packages/kotti-table/src/logic/column.js

```diff
@@ -90,7 +105,11 @@ export function setColumn(state, { column, index, deleted }) {
 		if (deleted) {
```

Comment:
(2) This may be right, but you set `_deleted` to false on both if and else, which seems in conflict the above statement.

Reply:
ok. nice one. I will set it in the setColumn call.

Reply:
actually, what is wrong with it?

Reply:
Discussed in person: It is trying to correct an initialization error happening elsewhere, but the culprit cannot be found at this moment and this needs to be revisited.